charts/opentelemetry-collector/UPGRADING.md

```diff
@@ -189,3 +194,16 @@ config:
       metrics: null
       logs: null
 ```
+
+### Set resources:
```

Comment:
Instead of its own section you can include this directly under the new `## 0.22.1 to 0.23.0` section